streaming-binlog/src/main/java/org/openmrs/analytics/FhirConverter.java

```diff
@@ -91,11 +98,17 @@ public void process(Exchange exchange) {
 			log.trace("Skipping disabled events ..." + table);
 			return;
 		}
-		if (payload.get("uuid") == null) {
-			log.error(String.format("No uuid column for table %s ignoring payload %s ", table, payload));
-			return;
+		
+		String uuid = "";
+		
+		if (payload.get("uuid") != null) {
+			uuid = payload.get("uuid").toString();
+		} else {
```

Comment:
Please add a unit-test in `FhirConverterTest` for this case of fetching `uuid` from the parent table.

Reply:
@bashir2 thanks for the review 

Am not sure why we need to cover this test case again in the `FhirConvertTest `since we covered the fetching of the uuid from the parent table in the` GetUuidUtilTest `, wont it be repetition of code 
I also did this as u hard earlier on requested that i can test the `getUuid(`) method directly to prove that it does whats its refrenced to do in the `FhirConvertTest `